helpers/githubHelper.js

```diff
@@ -1,52 +1,100 @@
 'use strict';
 let superAgent = require('superagent');
-let config = require('../ludwig-conf');
+const githubConfig = {
+	referencesEndpoint:'https://api.github.com/repos/hoshin/git-api-tests/git/refs',
+	createContent:'https://api.github.com/repos/hoshin/git-api-tests/contents/',
+	createPullRequest:'https://api.github.com/repos/hoshin/git-api-tests/pulls'
```

Comment:
I thought that's what we agreed upon for github config that'll probably never move, but sure I can extract it in a specific config file =)